src/application/metrics.rs

```diff
@@ -18,14 +18,16 @@ pub fn get_metrics() -> String {
 ///
 /// Panic if failed to initialise
 pub fn init_metrics() {
+    register_metrics();
+
     metrics::set_recorder(&*RECORDER).expect("telemetry: Can't set recorder.");
```

Comment:
is there a reason why we set metrics firstly and then call to fallible function which will break execution ?